internal/bminventory/inventory.go

```diff
@@ -76,7 +76,7 @@ import (
 const DefaultUser = "kubeadmin"
 
 const WindowBetweenRequestsInSeconds = 10 * time.Second
-const mediaDisconnectionMessage = "Cannot read from the media (ISO) - media was likely disconnected"
+const mediaDisconnectionMessage = "Unable to read from the discovery media.  It was either disconnected, or poor network conditions prevented it from being read.  Try using the minimal ISO option, and be sure to keep the media connected until the installation completes"
```

Comment:
There are double-whitespaces in the message. Is this by design?

Reply:
@avishayt

Reply:
Just remove the double spaces :)